include/drivers/gpio.h

```diff
@@ -451,6 +457,21 @@ struct gpio_callback {
  *
  * For internal use only, skip these in public documentation.
  */
+
+/* Used by driver api function pin_interrupt_configure, these are defined
+ * in terms of the public int flags so we can just mask and pass them
+ * through to the driver api
+ */
+enum gpio_int_trigger_mode {
```

Comment:
The enum name should match the name of its values, i.e. we should either rename the enum to `enum gpio_int_mode` or use `GPIO_INT_TRIGGER_MODE_` as a prefix for the enum values.

Reply:
Changed enum name.